lib/child_process.js

```diff
@@ -97,6 +99,65 @@ const MAX_BUFFER = 1024 * 1024;
 const isZOS = process.platform === 'os390';
 let addAbortListener;
 
+// Enhanced ChildProcess class with Windows encoding support
+class ChildProcess extends OriginalChildProcess {
```

Comment:
Introducing a class just to "highjack" spawnings of `cmd.exe` on Windows doesn't feel right